phi/playground/routes.py

```diff
@@ -0,0 +1,134 @@
+import base64
+from typing import List, Optional, Generator, Dict, cast, Union
+
+from fastapi import APIRouter, HTTPException, UploadFile
+from fastapi.responses import StreamingResponse, JSONResponse
+
+from phi.agent.agent import Agent, RunResponse
+from phi.agent.session import AgentSession
+from phi.playground.operator import format_tools, get_agent_by_id, get_session_title
+from phi.utils.log import logger
+
+from .schemas import (
```

Comment:
@manthanguptaa here we should do `phi.playground.schemas`

Reply:
nice catch